client/my-sites/sidebar/index.jsx

```diff
@@ -45,8 +49,11 @@ import {
 	canCurrentUserUseEarn,
 	canCurrentUserUseStore,
 } from 'state/sites/selectors';
+import getSiteChecklist from 'state/selectors/get-site-checklist';
+import getSiteTaskList from 'state/selectors/get-site-task-list';
 import canCurrentUserUseCustomerHome from 'state/sites/selectors/can-current-user-use-customer-home';
 import canCurrentUserManagePlugins from 'state/selectors/can-current-user-manage-plugins';
+import { getCompletionStatus } from 'lib/checklist';
```

Comment:
I'm stuck on why this is causing a Lint error - it is used and it is necessary. Any ideas?

Reply:
It is unused, the usage in that file is `const { completed, total } = siteTasklist.getCompletionStatus();`

Reply:
I think we can simply drop this line, unless @mmtr you had suggestions on how to better fetch tasks.

Note that we may want future follow ups for modifying APIs if we'll need this information on most page loads.

Reply:
> I think we can drop this line, unless @mmtr you had suggestions on how to better fetch tasks.

Yeah, this is the better way of handling it at the moment so we need to remove this import since it's unused.

Reply:
Ah, oops, you're right! Fixed. :)